androidapp/Crisislab/app/src/main/kotlin/com/example/crisislab/MainActivity.kt

```diff
@@ -17,68 +17,80 @@ import okhttp3.Request
 import okhttp3.WebSocket
 
 class MainActivity() : ComponentActivity() {
+    // ViewModels for managing logs and socket status
     lateinit var logViewModel: LogViewModel
-    lateinit var socketStatusViewModel: SocketStatusViewModel;
-    private lateinit var binding:ActivityMainBinding
-    val notificationModule: NotificationModule = NotificationModule;
-    lateinit var notificationHandler: NotificationHandler;
+    lateinit var socketStatusViewModel: SocketStatusViewModel
+    // Binding object for the main activity layout
+    private lateinit var binding: ActivityMainBinding
+    // Notification module instance
+    val notificationModule: NotificationModule = NotificationModule
+    // Notification handler instance
+    lateinit var notificationHandler: NotificationHandler
```

Comment:
Hmm I wonder what this thing called a notificationHandler is

Reply:
Fixed, will push soon